### DISCUSSION TITLE
Can't connect to debian server remotely (only using localhost:8889)

### DISCUSSION THREAD

**Discussion by kwcwaf** [Velocidex/velociraptor#1525] [Q&A] [answered]
Everything is running properly, connected a client to the server and can access the Velociraptor console on the server using https://localhost:8889 with no problems. 

But I cannot connect from remote server (only localhost). The DNS is setup properly and clients are finding it ok using DNS. Any ideas on why its not accepting remote connections and giving "ERR_CONNECTION_REFUSED"? Using self-signed certificate option in the server.config.yaml.

Thanks for any help on this.

**Comment by scudette** [comment, marked as answer]
When using self signed mode the GUI listens on local host only for security
reasons.

We can open it by changing GUI.bind_address if you accept the security
risk. It's better to use ssh tunneling though to protect the connection
instead.

Thanks
Mike

**Reply by kwcwaf** [reply]
Thanks scudette. That was the issue.
